README.md

```diff
@@ -1,10 +1,18 @@
 # Software Engineering: Homework-2b
 
-Write a "good" Github Repo
-
 ## Description
 
-To be submitted as homework assignment HW-2b for the course Software Engineering (CSC-510) under Professor Timothy Menzies.
+The primary purpose of this repository was to get familiar with the various features of Github and to concisely integrate them. This repository was created as a homework assignment (HW-2b) for our Software Engineering (CSC-510) course under [Prof. Timothy Menzies](http://menzies.us/).
```

Comment:
This is a good description, however it describes in a verbose way what our 'simple' goal is for our HW-2b, which is to create a good repo. It is good, but should be added as a second point, because we do not intend to obfuscate, even unintentionally.